README.md

```diff
@@ -112,8 +112,8 @@ Please report bugs and suggestions on [GitHub][github]. Thank you!
 
 [license]: https://www.census.gov/data/software/x13as/disclaimer.html "License Information and Disclaimer"
 
-[examples]: http://www.seasonal.website/examples.html "Examples of X-13ARIMA-SEATS in R"
+[examples]: https://www.seasonal.website/examples.html "Examples of X-13ARIMA-SEATS in R"
```

Comment:
Is this one bothering you as well? I know I should change this but https doesn't work yet. The alternative would be to remove the links.

Reply:
No, apologies, I "mechanically" switched without (ahem!) testing.  Please feel free to revert to http then.  We can remove if CRAN complains.

Reply:
Now done.